src/impl.ml

```diff
@@ -178,7 +177,8 @@ let machine_progress_bar total_work =
 
 let no_progress_bar _ _ = ()
 
-let stream_human common _ s _ _ ?(progress = no_progress_bar) () =
+let [@warning "-27"]
+  stream_human _common _ s _ _ ?(progress = no_progress_bar) () =
```

Comment:
I spent a while experimenting with this a while ago, an alternative to silence warnings for labeled arguments is to use `?label:_` (note: no parenthesis, although this makes the type `'a` and it might be incompatible with the mli), so something like this might work, but not sure: `let stream_human common _ s _ _ ?progress:_  ()`